### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SYK-08**
* Resolves: Grammatical errors in README.md

## Summary
Improved grammatical mistakes in README.md to improve readability for users.

## TODO

- [ ] ...

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Otherwise, git checkouts can be handled the same as release archives, by using t
 - [🌊 WAVE](https://wave.webaim.org/extension/) for accessibility testing
 - [🚨 Lighthouse](https://developers.google.com/web/tools/lighthouse/) for testing performance, accessibility, and more
 
-#### Helpful bots at github :robot:
+#### Helpful bots at gitHub :robot:
```
Comment (susnux) [inline README.md:67]:
```suggestion
#### Helpful bots at GitHub :robot:
```

**Comment by susnux** [comment]
Thank you for you contribution, I have only one comment about the `GitHub` spelling, see comment.

**Comment by SYK-08** [comment]
I have resolved the comment. Thank you for pointing it out.

**Comment by susnux** [comment]
> I have resolved the comment. Thank you for pointing it out.

Please push the changes :)

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**